installer/dockerfile/controller-manager/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.21.10 AS builder
+FROM golang:1.22.0 AS builder
```

Comment:
It is recommended to use a version with a higher minor version number. Such as 1.22.9

Reply:
thanks!